latex/ledger-spec.tex

```diff
@@ -218,13 +220,19 @@ \section{Notation}\label{sec:notation}
     & = \{ k \mapsto v \mid k \mapsto v \in \var{map}, ~ v \notin \var{set} \}
     & \text{range exclusion}
     \\
-    A \unionoverrideRight B
-    & = (\dom B \subtractdom A)\cup B
+    M \unionoverrideRight N
+    & = (\dom N \subtractdom M)\cup N
     & \text{union override right}
     \\
-    A \unionoverrideLeft B
-    & = A \cup (\dom A \subtractdom B)
+    M \unionoverrideLeft N
+    & = M \cup (\dom M \subtractdom N)
     & \text{union override left}
+    \\
+    M \unionoverridePlus N
+    & = (M \triangle N)
```

Comment:
What is this operator? Domain restriction that drunk too much and now lies face up? :D

Reply:
something like that :)  I was using that symbol for the "symmetric difference" (ie `(M cup N)\(M cap N)`), and hoping that it was standard enough to be recognizable to all, but perhaps it's not. Would the circle with a line be better? Or perhaps just and explanation somewhere.

Reply:
Now that you mentioned it I remember seeing "symmetric difference" back in school. Maybe we can include the one line definition you gave above, for people with bad memory like me :)

Reply:
perfect! :+1: